carmen-salas/short-response.md

```diff
@@ -0,0 +1,79 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+* This will log ‘My favorite singer is Omar Apollo’ because the variable is declared with the string ‘Omar Apollo’ and is being concatenated with the string “my favorite singer is”
```

Comment:
Incorrect. We declare the variable `singer` with the keyword `let` **inside of a block**. What do we know about variables declared with `let`?